pkg/local_object_storage/shard/errors.go

```diff
@@ -10,19 +10,19 @@ import (
 // IsErrNotFound checks if error returned by Shard Get/Head/GetRange method
 // corresponds to missing object.
 func IsErrNotFound(err error) bool {
-	return errors.As(err, new(apistatus.ObjectNotFound))
+	return errors.Is(err, apistatus.ErrObjectNotFound)
```

Comment:
how about unwrap them now?

the only thing that could stop me is that returning status errors is pretty weird for local layer, and it would be worth replacing them with more generic ones. These functions would reduce the amount of changes in this case

Reply:
sorry, did not get you. what exactly to unwrap?

Reply:
funcs in this file. But i saw an issue (urs as i remember) which proposes to replace API errors with more generic ones. Sad i cannot find it anymore. With it, it's better to keep the funcs for now